src/MAUI/Maui.Samples/Samples/UtilityNetwork/TraceUtilityNetwork/TraceUtilityNetwork.xaml

```diff
@@ -1,90 +1,95 @@
-﻿<ContentPage x:Class="ArcGISRuntime.Samples.TraceUtilityNetwork.TraceUtilityNetwork"
-             xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
-             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
-             xmlns:esriUI="clr-namespace:Esri.ArcGISRuntime.Maui;assembly=Esri.ArcGISRuntime.Maui">
-    <Grid>
-        <Grid.RowDefinitions>
-            <RowDefinition Height="auto" />
-            <RowDefinition Height="*" />
-            <RowDefinition Height="auto" />
-        </Grid.RowDefinitions>
-        <esriUI:MapView x:Name="MyMapView"
-                        Grid.Row="0"
-                        Grid.RowSpan="2"
-                        GeoViewTapped="OnGeoViewTapped" />
-        <Grid x:Name="StatusGrid">
-            <Label x:Name="Status"
-                   Grid.Row="0"
-                   Margin="5"
-                   HorizontalOptions="FillAndExpand"
-                   Text="Loading sample..."
-                   TextColor="White" />
-        </Grid>
-        <StackLayout x:Name="BusyIndicator"
-                     Grid.Row="0"
-                     Grid.RowSpan="2"
-                     HeightRequest="40"
-                     IsVisible="False"
-                     VerticalOptions="Center">
-            <ActivityIndicator HeightRequest="40"
-                               HorizontalOptions="Center"
-                               IsRunning="True"
-                               VerticalOptions="Center">
-                <!--<ActivityIndicator.Color>
-                    <OnPlatform x:TypeArguments="Color" iOS="#2499CE" />
-                </ActivityIndicator.Color>-->
-            </ActivityIndicator>
+﻿<ContentPage
+    x:Class="ArcGISRuntime.Samples.TraceUtilityNetwork.TraceUtilityNetwork"
+    xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+    xmlns:esriUI="clr-namespace:Esri.ArcGISRuntime.Maui;assembly=Esri.ArcGISRuntime.Maui">
+    <Grid Style="{DynamicResource EsriSampleContainer}">
+        <esriUI:MapView
+            x:Name="MyMapView"
+            GeoViewTapped="OnGeoViewTapped"
+            Style="{DynamicResource EsriSampleGeoView}" />
+        <StackLayout
```

Comment:
This is not centered in the screen. Set ColumnSpan and RowSpan values.

Reply:
On the StackLayout or the GeoView?